doc/api/process.md

```diff
@@ -1807,11 +1807,12 @@ This function is only available on POSIX platforms (i.e. not Windows or
 Android).
 This feature is not available in [`Worker`][] threads.
 
-## process.setUncaughtExceptionCaptureCallback(fn)
+## process.setUncaughtExceptionCaptureCallback(owner, fn)
```

Comment:
A bit concerned about backwards compat here. Is the new `owner` argument optional? What about existing code that may be calling `setUncaughtExceptionCaptureCallback(fn)`?

Reply:
why not use a Set instead of a Map and use the functions themselves as keys

Reply:
@devsnek's suggestion seems like a good alternative, I'll try that 👍